### DISCUSSION TITLE
Is there a way to disable stepsFromCommands?

### DISCUSSION THREAD

**Discussion by Truput** [allure-framework/allure2#2722] [Questions & Support] [answered]
Previously we used shelex plugin, but we need to switch to a newer version of cypress. Using allure-cypress plugin makes the report less clean and readable with all those stepsFromCommands, as we wrap all our functions with steps

**Comment by delatrie** [comment, marked as answer]
Currently, there is no such way. Here is a feature request to track: https://github.com/allure-framework/allure-js/issues/1129

**Comment by Truput** [comment]
@delatrie got it, thanks
